sdk/src/accounts/pollingDriftClientAccountSubscriber.ts

```diff
@@ -528,24 +534,25 @@ export class PollingDriftClientAccountSubscriber
 	}
 
 	public getOraclePriceDataAndSlot(
-		oraclePublicKey: PublicKey
+		oraclePublicKey: string
```

Comment:
can maintain backwards compatibility if accept both types and check for something like `oraclePublicKey['toBase58'] !== undefined`